### ISSUE TITLE
Bug: Crash when map doesn't have [TimingPoints]

### ISSUE THREAD

**Issue by richardscull** [kionell/osu-classes#8]
When map doesn't have [TimingPoints] field, it crashes with this error

```
\osu-classes\lib\index.cjs:3476
    return 60000 / Number(entries[0][0]) * this.difficulty.clockRate;
                                    ^

TypeError: Cannot read properties of undefined (reading '0')
```

You can create any map without BPM and it supposed to crash. 
Happened to me on this [map](https://osu.ppy.sh/beatmapsets/1002686#osu/2133911), zuozhewan diff.

**Comment by kionell** [comment]
Yeah, it's a known bug and will be fixed with next release.

**Comment by kionell** [comment]
Fixed in #9
